autosklearn/util/logging_.py

```diff
@@ -7,18 +7,24 @@
 import yaml
 
 
-def setup_logger(output_file=None):
-    with open(os.path.join(os.path.dirname(__file__), 'logging.yaml'),
-              'r') as fh:
-        config = yaml.load(fh)
-    if output_file is not None:
-        config['handlers']['file_handler']['filename'] = output_file
+def setup_logger(output_file=None, log_config=None):
+    # log_config must be a dictionary object specifying the configuration for
+    # the loggers to be used in auto-sklearn.
+    if log_config is not None:
+        config = log_config
+        if output_file is not None:
```

Comment:
I don't think these two lines should be here. They would basically revert the choices of the user, right?